ci_stuff/git_hooks/pre-commit

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+git diff --cached --name-only --diff-filter=ACMRT |
```

Comment:
This is copy paste. Avoid it. Please, remove all checks from pre-commit. Imagine you will need to add some additional checker. With current approach youll need to update precommit and all users will have to update precommit with new one manually. This is error prone. The way to go is to create some script with all checks and from pre commit hook just call this script.